crap-advisor-frontend/src/core/components/Map/index.jsx

```diff
@@ -2,34 +2,32 @@ import React, {useEffect, useState} from "react";
 import "./index.css"
 import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
 import RestroomMarker from "../RestroomMarker";
+import {apiUrl} from "../../../app/constants";
 
-const Map = () => {
-    const apiUrl = "http://localhost:8082"
+const Map = ({selectRestroom}) => {
     const defaultPosition = [53.21176, 50.18394]
     const defaultZoom = 13
     const [restrooms, setRestrooms] = useState([])
 
     useEffect(() => {
         fetch(apiUrl + "/restroom/all")
-        .then((response) => response.json())
-        .then((data) => {
-            setRestrooms(data)
-            console.log(`Received ${data.length} restrooms`)
-        })
+            .then((response) => response.json())
+            .then((data) => {
+                setRestrooms(data)
+                console.log(`Received ${data.length} restrooms`)
+            })
     }, [])
 
     return (
         <MapContainer className="map" center={defaultPosition} zoom={defaultZoom} zoomControl={false}>
             <TileLayer
-                attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+                attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a>'
                 url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
             />
             <ZoomControl position="bottomright"/>
             {
                 restrooms.map((restroom, i) => {
-                    return (
-                        <RestroomMarker key={i} restroom={restroom}/>
-                    )
+                    return (<RestroomMarker key={i} restroom={restroom} selectRestroom={selectRestroom}/>)
```

Comment:
You can remove round braksets if your return value contains only one component

Reply:
Good tip!